### PR TITLE
Merge development of web interface into repository

### PR DESCRIPTION
**Pull request by nicotom**

### CODE REVIEW

File: tracker_lib.py [hunk 1]
```diff
@@ -31,10 +37,11 @@ def nothing(a):
  
 #while not key == ord('q'):
 TH = cv2.getTrackbarPos('TH','threshold')        ### gustavo 
-upper_white = np.array([TH], dtype=np.uint8)  ### gustavo 
-arduino = serial.Serial('/dev/ttyACM0', 9600) ### gustavo
+upper_white = np.array([TH], dtype=np.uint8)  ### gustavo
+# arduino = serial.Serial('/dev/ttyACM0', 9600) ### gustavo
```
Comment (facu-s) [inline tracker_lib.py]: This line must be un-commented. It was commented at the beginning only for debug purposes.

File: tracker_lib.py [hunk 2]
```diff
@@ -119,40 +126,53 @@ def sequence_test():
...
+    print("Press 2 to exit, 3 to stop, 4 to continue")
     try:
         camera = PiCamera()
 #        camera.roi (0.5,0.5,0.25,0.25)
-	camera.resolution = SIZE
         stream = PiRGBArray(camera, size=SIZE)
-        time.sleep(0.1)  # allow the camera to warmup
+        camera,stream = camera_attr(camera,stream)
+
+        # stream = PiRGBArray(camera, size=SIZE)
```
Comment (facu-s) [inline tracker_lib.py]: These comments, should be deleted.

File: tracker_lib.py [hunk 3]
```diff
@@ -190,20 +210,19 @@ def wait_key():
...
-    cv2.line(image, (SIZE[0]/2, 0), (SIZE[0]/2, SIZE[1]), (255, 0, 0), 1)
-#    cv2.putText(image, "A+", (SIZE[0]/4, SIZE[1]/4), cv2.FONT_HERSHEY_SIMPLEX, 1, (80, 80, 200))
-#    cv2.putText(image, "B+", (3 * SIZE[0]/4, SIZE[1]/4), cv2.FONT_HERSHEY_SIMPLEX, 1, (80, 80, 200))
-#    cv2.putText(image, "A-", (SIZE[0]/4, 3 * SIZE[1]/4), cv2.FONT_HERSHEY_SIMPLEX, 1, (80, 80, 200))
-#    cv2.putText(image, "B-", (3 * SIZE[0]/4, 3 * SIZE[1]/4), cv2.FONT_HERSHEY_SIMPLEX, 1, (80, 80, 200))
-    cv2.circle(image, (SIZE[0]/2, SIZE[1]/2), CENTER_RADIUS, (80, 80, 200), 1)
+
+    cv2.line(image, (0, int(SIZE[1]/2)), (int(SIZE[0]), int(SIZE[1]/2)), (255, 0, 0), 1)
+    cv2.line(image, (int(SIZE[0]/2), 0), (int(SIZE[0]/2), int(SIZE[1])), (255, 0, 0), 1)
+    # cv2.putText(image, "A+", (int(SIZE[0]/4), int(SIZE[1]/4)), cv2.FONT_HERSHEY_SIMPLEX, 1, (80, 80, 200))
```
Comment (facu-s) [inline tracker_lib.py]: Since we don't use this anymore, these comments should also be deleted.

File: tracker_lib.py [hunk 4]
```diff
@@ -236,15 +255,16 @@ def check_quadrant(cx, cy):
     Red shows that the camera is centered.
     """
     global available_leds
-    returnString ="{} , {}".format(cx,cy)  ### gustavo
-#    returnString1 ="cy{}".format(cy) 
+    # returnString ="{} , {}".format(cx,cy)  ### gustavo
```
Comment (facu-s) [inline tracker_lib.py]: We should un-comment this line too.

File: tracker_lib.py [hunk 5]
```diff
@@ -236,15 +255,16 @@ def check_quadrant(cx, cy):
...
+#    returnString1 ="cy{}".format(cy)
```
Comment (facu-s) [inline tracker_lib.py]: Also this one.

File: tracker_lib.py [hunk 6]
```diff
@@ -236,15 +255,16 @@ def check_quadrant(cx, cy):
...
 #    print cx,cy
     result = ""
 #    print(returString1)
-    print(returnString)                    ### gustavo
-
-    arduino.write(returnString + '\n')    ### gustavo
+    # print(returnString)                    ### gustavo
 
+    # arduino.write(returnString + '\n')    ### gustavo
```
Comment (facu-s) [inline tracker_lib.py]: Same applies here.

File: tracker_lib.py [hunk 7]
```diff
@@ -300,8 +320,12 @@ def obtain_single_contour(b_frame):
     Obtain the x and y coordinates of a single contour.
     When none is found, it returns: (-1, -1)
     """
-    contours, _h = cv2.findContours(b_frame, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
-    cx, cy = (90,90)  # When none is found, a negative coordinates are returned.
+    try:
+        contours, _h = cv2.findContours(b_frame, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+    except ValueError:
+        _, contours, _h = cv2.findContours(b_frame, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+    # cx, cy = (90, 90)  # NOTE: This was replaced with negative coordinates
```
Comment (facu-s) [inline tracker_lib.py]: Delete comment.

**Review by facu-s — CHANGES_REQUESTED** [review]
Minor changes required. All the rest is good.

**Comment by nicotom** [comment]
@facu-s fixed, feel free to merge it.

**Pull request merged**